frontend/src/sections/EventSection/EventCard.tsx

```diff
@@ -66,12 +65,14 @@ const EventCard: React.FC<EventProps> = ({
             <span className='font-bold'>Mode: </span>
             {event_mode}
           </p>
-          {venue && <p className='text-gray-700 mb-2 line-clamp-4'>
-            <span className='font-bold'>Venue: </span>
-            <br />
-            {venue}
-          </p>}
-          <p className='text-gray-600'>{dayjs(time).format('hh:mm A')}</p>
+          {venue && (
+            <p className='text-gray-700 mb-2 line-clamp-4'>
+              <span className='font-bold'>Venue: </span>
+              <br />
+              {venue}
+            </p>
+          )}
+          {/* <p className='text-gray-600'>{dayjs(time).format('hh:mm A')}</p> */}
```

Comment:
This creates some hydration error so commented for now until i investigate on the issue